financial-templates-lib/price-feed/CreatePriceFeed.js

```diff
@@ -0,0 +1,126 @@
+const { MedianizerPriceFeed } = require("./MedianizerPriceFeed");
+const { CryptoWatchPriceFeed } = require("./CryptoWatchPriceFeed");
+const { UniswapPriceFeed } = require("./UniswapPriceFeed");
+
+const Uniswap = require("../../core/build/contracts/Uniswap.json");
+
+async function createPriceFeed(web3, logger, networker, getTime, config) {
+  if (config.type === "cryptowatch") {
+    const requiredFields = ["apiKey", "exchange", "pair", "lookback", "minTimeBetweenUpdates"];
+
+    if (isMissingField(config, requiredFields, logger)) {
```

Comment:
Could we fail a bit more gracefully if this is missing? something like a `console.error` perhaps?

Reply:
`isMissingField()` method calls `logger.error()` in the case that it returns true -- I'm assuming that's sufficient?

Reply:
Discussed IRL -- we are on the same page here.